### ISSUE TITLE
Align Xen installation between Yocto and native

### ISSUE THREAD

**Issue by andr2000** [xen-troops/meta-xt-images#12]
The way build artifacts are installed using Xen makefile and Yocto build differ, which makes it not so convenient to use SDK and daily build.
Recipe should be adjusted to match *make install* from Xen.

**Comment by andr2000** [comment]
Xen's native make install:
```
lrwxrwxrwx  1 root root     10 Aug  9 14:42 xen -> xen-4.9-rc
lrwxrwxrwx  1 root root     10 Aug  9 14:42 xen-4 -> xen-4.9-rc
lrwxrwxrwx  1 root root     10 Aug  9 14:42 xen-4.9 -> xen-4.9-rc
-rw-r--r--  1 root root 754000 Aug  9 14:42 xen-4.9-rc
-rw-r--r--  1 root root   1544 Aug  9 14:42 xen-4.9-rc.config
-rw-r--r--  1 root root 754064 Aug  9 14:42 xen-uImage
-rw-r--r--  1 root root  10306 Aug  9 14:42 xenpolicy-4.9-rc

```

**Comment by iartemenko** [comment]
Fixed-in: xen-troops/meta-xt-prod-devel 49970f62a58cae83db9113e128602b4842e9bc3c
